### PR TITLE
Feature/accept invite

### PR DESCRIPTION
**Pull request by mikahpinegar**
- Invite user to technicians team
- Accept invite to technicians team
- Refactor calls to azure function to be cleaner and move getTokenSilently call into a function
- Remove tatus file that got added accidentally
- Increase snackbar timeout to maximum (10 seconds)
- Put secret in environment variable in azure function
- Add logging to tech-dashboard database azure-calls table, including os/browser info and function response

### CODE REVIEW

File: src/Azure/Functions/GithubIssues/GithubIssues/__init__.py [hunk 1]
```diff
@@ -24,7 +25,7 @@ def __init__(self, user):
     # method to fetch the Auth0 management API token	
     def get_token(self):	
         # payload and header	
-        auth0_payload = "{\"client_id\":\"RKkiHSDtTGQWijgy7q7Pi7sheonzWZke\",\"client_secret\":\"pzoI_Zwp4i-PSlb0lHVOZTyl8zZ8iQmuzJ8GNfR4pY1Ttr1PgXPK3Bbk-LdRXyf9\",\"audience\":\"https://psa-tech-dashboard.auth0.com/api/v2/\",\"grant_type\":\"client_credentials\"}"
```
Comment (rbandooni) [inline src/Azure/Functions/GithubIssues/GithubIssues/__init__.py:27]: Is it possible to add these `secrets` outside Github?

Comment (mikahpinegar) [reply src/Azure/Functions/GithubIssues/GithubIssues/__init__.py:27]: that is a good point. it should be possible

**Comment by brianwdavis** [comment]
Can we auto add users to this team: https://github.com/orgs/precision-sustainable-ag/teams/technicians/members

Then they automatically get access to the comments repo.

**Comment by mikahpinegar** [comment]
yes, that should be possible. will accept my request to join the team? i need to be a member to find the team id

**Comment by brianwdavis** [comment]
You have been added

File: src/Azure/Functions/GithubIssues/GithubIssues/__init__.py (hunk 1 above)
Comment (mikahpinegar) [reply src/Azure/Functions/GithubIssues/GithubIssues/__init__.py:27]: moved to env variable

**Pull request merged**